dockerfiles/init/modules/openshift/files/scripts/ocp.sh

```diff
@@ -171,12 +212,14 @@ server_is_booted() {
 
 wait_until_server_is_booted() {
   SERVER_BOOT_TIMEOUT=300
-  echo "[CHE] wait CHE pod booting..."
+  echo -n "[CHE] wait CHE pod booting..."
```

Comment:
Can you elaborate on why you did change the output here and how it would look like after the change?

Reply:
I allows the use of periods to indicate the script is working. It's similar to what has been done in other parts of the script.

```
[CHE] wait CHE pod booting...
[CHE] wait CHE pod booting....
[CHE] wait CHE pod booting......
```

Reply:
Are you sure that output will be 
`[CHE] wait CHE pod booting......`
not 
```
[CHE] wait CHE pod booting...
.
.
.
```
?

Reply:
Oh, I see that I missed what `-n` does. Sorry for confusing.